docs/deploy-controller.md

```diff
@@ -0,0 +1,100 @@
+# Deploying the CSI-Addons Controller
+
+The CSI-Addons Controller can be deployed by different ways:
+
+## Installation for versioned deployments
+
+The CSI-Addons Controller can also be installed  using the yaml files in `deploy/controller`.
+The versioned deployment is possible with the yaml files that get generated for a release,
+like [release-v0.3.0](https://github.com/csi-addons/kubernetes-csi-addons/releases/tag/v0.3.0).
```

Comment:
mention to use the latest release `https://github.com/csi-addons/kubernetes-csi-addons/releases/latest`